accounts-base/src/main/java/pm/gnosis/svalinn/accounts/base/repositories/AccountsRepository.kt

```diff
@@ -10,7 +10,7 @@ import pm.gnosis.svalinn.accounts.base.models.Signature
 interface AccountsRepository {
     fun loadActiveAccount(): Single<Account>
 
-    fun signTransaction(transaction: Transaction): Single<String>
+    fun signTransaction(transaction: Transaction.Legacy): Single<String>
```

Comment:
will this break existing library clients?

Reply:
No, because they could still use previous version of the library. Also we have agreed not to make this library public - that's why it's in 5afe space and not in safe-global. Currently it is for internal use only.